.github/workflows/android_build.yml

```
@@ -0,0 +1,31 @@
+name: Android Build
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set Up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
+
+      - name: Create file
+        run: cat /home/runner/work/<Project-Name>/<Project-Name>/app/google-services.json | base64
+
+      - name: Decode google-services.json
+        env:
+          FIREBASE_SECRET: ${{ secrets.FIREBASE_SECRET }}
+        run: echo $FIREBASE_SECRET > /home/runner/work/MeuBeats/MeuBeats/app/google-services.json
```

Estes dois passou poderia ser apenas um, pois o `echo` já ira escrever seu arquivo se o mesmo não existir.

```suggestion
      - name: Decode google-services.json
        env:
          FIREBASE_SECRET: ${{ secrets.FIREBASE_SECRET }}
        run: echo $FIREBASE_SECRET > app/google-services.json
```